spiffworkflow-frontend/src/ContainerForExtensions.tsx

```
@@ -144,35 +150,66 @@ export default function ContainerForExtensions() {
-              }
-            } catch (jsonParseError: any) {
-              console.error(
-                `Unable to get navigation items for ${processModel.id}`,
+      const eni: UiSchemaUxElement[] = [];
+      const cssFiles: Array<{ content: string; id: string }> = [];
+
+      processModels.forEach((processModel: ProcessModel) => {
+        const extensionUiSchemaFile = processModel.files.find(
+          (file: ProcessFile) => file.name === 'extension_uischema.json',
+        );
+        if (extensionUiSchemaFile && extensionUiSchemaFile.file_contents) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)